WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteCreationRotatingMessageView.swift

```diff
@@ -0,0 +1,189 @@
+import UIKit
+
+class SiteCreationRotatingMessageView: UIView {
+    private struct Margins {
+        static let horizontalMargin: CGFloat = 0
+        static let verticalMargin: CGFloat = 0
+
+        static let horizontalSpacing: CGFloat = 15
+    }
+
+    private struct Parameters {
+        //Animation parameters
+        static let transitionAnimationDuration = TimeInterval(0.5)
+        private static let statusDisplayDuration = TimeInterval(2)
+
+        //Adjust the total display time to take the fade into consideration
+        static var totalStatusDisplayDuration: TimeInterval {
+            return statusDisplayDuration + transitionAnimationDuration
+        }
+    }
+
+    /// An array of status messages to rotating through
+    private(set) var statusMessages: [String]!
+
+    // MARK: - State Management
+
+    /// A timer that determines how long to show each message
+    internal var animationTimer: Timer!
```

Comment:
Out of curiosity, any special reason to explicit declare the `internal` access control?

I guess we can change this one and `visibleIndex` to `private(set)`. What do you think?

Reply:
No specified reason other than an oversight on my part. I've changed both of those to `private(set)`